scripts/cloud-release.sh

```diff
@@ -6,7 +6,8 @@ export RELEASE_PROJECT_NAME=amplify-category-api-release-workflow
 export DEPRECATE_PROJECT_NAME=amplify-category-api-deprecate-workflow
 
 function triggerRelease {
-  triggerProjectBatch $RELEASE_ACCOUNT_PROD $RELEASE_ROLE_NAME "${RELEASE_PROFILE_NAME}Prod" $RELEASE_PROJECT_NAME "release"
+  branch_name=$(git branch --show-current)
```

Comment:
Since this will only be used for prod releases for Gen2 or LTS, can we add some validation here that it's one of our expected branch names. That will prevent us from accidentally triggering a release on a PR branch, for example.

Reply:
Added a check to limit this to release branches. The reason not to be too strict here is to save us from making changes in multiple places (or forgetting to) when we add more release branches, since we already have publish scripts for only specific branches defined in our outer `package.json`.